Cucumber/features/project1/pages/top_part.rb

```diff
@@ -0,0 +1,22 @@
+module TopPart
+  attr_accessor :logout_btn,:home_btn
+
+  def initialize(driver)
+    @driver = driver
+    @logout_btn = @driver.find_element(css: '.logout>a')
+  end
+
+  def logout_btn_click
+    @logout_btn.click
+  end
+  
+  def home_btn
+    @driver.find_element(xpath: "//a[contains(@href,'home')]")
+  end
```

Comment:
believe this should be in initialize(according to approach with attribution_accessor? still don't like it though). I mean why do we make separate reader method, when we are already making attribute accessor method for this variable?